doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,148 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given corresponding source and target control points, TPS is used to transform a space.
+In this case our space is a 2D image.
+
+This script shows how to use the Thine Plate Spine Transformation for
+image warping.
+
+For further information on TPS Transformation, see:
+
+.. [1] Wikipedia, Thin plate spline
+       https://en.wikipedia.org/wiki/Thin_plate_spline
+
+.. [2] Weisstein, Eric W. "Biharmonic Equation." From MathWorld--A Wolfram Web Resource
+       https://mathworld.wolfram.com/BiharmonicEquation.html
+
+.. [3] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+       decomposition of deformations." IEEE Transactions on pattern analysis and
+       machine intelligence 11.6 (1989): 567–585.
+
+
+Image Deformation
+=================
+
+In this example we will see how to use the thin plates spline interpolation to deform an image.
+To deform the image, the displacement of every pixel is needed.
+In our image, we define 6 source and destination points labelled 1-6: 1-4 in the image
+corners, 5 near the smile corner and 6 in an eye. At the corners, there are no
```

Comment:
```suggestion
corners, "5" near the smile corner and "6" in an eye. At the corners, there are no
```
Because this "6" has nothing to do with the one in "6 source and destination points"